engine/components-api/src/main/java/com/cloud/resource/ResourceManager.java

```diff
@@ -47,11 +47,6 @@
  */
 public interface ResourceManager extends ResourceService, Configurable {
 
-    ConfigKey<Integer> HostMaintenanceRetries = new ConfigKey<>("Advanced", Integer.class,
```

Comment:
@anuragaw @nvazquez I thought we wanted to have a global setting to define max retries? Can you check the requirement and the implementation regarding this, or if any other setting is used instead?

Reply:
Saw MaxRetries usage later in the code, in this case if we're deprecating/removing this ConfigKey, please add an upgrade path to delete the configuration from cloud.configuration as well.

Reply:
Good point. Will do.

Reply:
Done.